options.go

```diff
@@ -1211,3 +1375,38 @@ func (opts *Options) Destroy() {
 	opts.env = nil
 	opts.bbto = nil
 }
+
+// GetTickerCount gets the current value of Ticker
+func (opts *Options) GetTickerCount(ticker Ticker) uint64 {
+	retval, _ := C.rocksdb_get_ticker_count(opts.c, C.uint(ticker))
+
+	return uint64(retval)
+}
+
+// GetAndResetTickerCount gets the current value of Ticker, and resets it to zero
+func (opts *Options) GetAndResetTickerCount(ticker Ticker) uint64 {
+	retval, _ := C.rocksdb_get_and_reset_ticker_count(opts.c, C.uint(ticker))
+
+	return uint64(retval)
+}
+
+// RecordTick adds count to the specified ticker
+func (opts *Options) RecordTick(ticker Ticker, count uint64) {
+	C.rocksdb_record_tick(opts.c, C.uint(ticker), C.ulonglong(count))
+}
+
+// SetTickerCount sets the current ticker count.
+func (opts *Options) SetTickerCount(ticker Ticker, count uint64) {
+	C.rocksdb_set_ticker_count(opts.c, C.uint(ticker), C.ulonglong(count))
+}
+
+// SetStatsLevel sets the current statistics logging level - See <rocksdb/statistics.h> for more information
+func (opts *Options) SetStatsLevel(sl StatsLevel) {
+	C.rocksdb_set_stats_level(opts.c, C.uchar(sl))
+}
+
+// GetStatsLevel retrieves the current statistics logging level - See <rocksdb/statistics.h> for more information
+func (opts *Options) GetStatsLevel() StatsLevel {
+	ret := C.rocksdb_get_stats_level(opts.c)
+    return StatsLevel(ret)
```

Comment:
Make sure the code is formatted with `go fmt` (I wish we had the CI setup for this repo).